app/models/user.rb

```diff
@@ -1,5 +1,16 @@
 class User < ApplicationRecord
   has_many :microposts, dependent: :destroy
+<<<<<<< HEAD
+=======
+  has_many :active_relationships, class_name: "Relationship",
+                                  foreign_key: "follower_id",
```

Comment:
```suggestion
                                  foreign_key: :follower_id,
```
Như này không được à em